### PR TITLE
WIP: work on release 2017.1

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
For @pdebuyl  to take over

### CODE REVIEW

**Comment by GaelVaroquaux** [comment]
Remaining work:

* [ ] Do the changelog.
* [ ] Check that all the instances of "2016" have been changed to "2017"
* [ ] Build using make install in an environment with the right set of package dependencies.

**Comment by pdebuyl** [comment]
@rougier @emmanuelle @gertingold @GaelVaroquaux @vahtras I put up a changelog here: https://github.com/pdebuyl/scipy-lecture-notes/blob/rel_2017/CHANGES.rst

I used the PR page from GitHub and the git log. I'll check that every contributor since 2016.1 is added to "authors" but for the manual changelog I'll welcome an extra set of eyes.

Target: release by tomorrow afternoon

**Comment by GaelVaroquaux** [comment]
Maybe you should do a PR to master, and I close this PR. To ease review.

**Comment by GaelVaroquaux** [comment]
Comment on the release notes:

* Vince Knight also worked on the requirement bullet point.
* I think that the new and updated chapters should go first: infrastructure is very important to us, developers, but it matters less to end users.

**Comment by rougier** [comment]
Small comment: maybe content update/change first  (most important)

**Comment by pdebuyl** [comment]
I admit that I just wrote the changes as I read about them and did not pay attention to the order.

The order is now more or less from most visible to least visible.

I fixed Vince Knight's contribution to the requirements/environment system.

**Comment by pdebuyl** [comment]
@emmanuelle, @GaelVaroquaux or @vahtras : I need push access to the website repository.

I am ready to push apart from that :-)

**Comment by GaelVaroquaux** [comment]
> @emmanuelle, @GaelVaroquaux or @vahtras : I need push access to the website repository.

I have added you the core team.

Looking forward to the release!

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,36 @@
 What's new
 ==========
 
+Release 2017.1 (September 2017)
```
Comment (GaelVaroquaux) [inline CHANGES.rst]: This should probably be changed to Octobre :)

**Comment by GaelVaroquaux** [comment]
I think that once you have changed "September" to "October", we should:

* [ ] merge this PR to master
* [ ] Tag
* [ ] Push the tag
* make install (careful about doing it in the right environment)

**Comment by pdebuyl** [comment]
Shall I tweet about the new http://www.scipy-lectures.org/ ?

**Comment by gertingold** [comment]
Great work! Thanks to everybody who contributed to the new edition.

**Comment by GaelVaroquaux** [comment]
Yes! 

Congratulations for the release! Thanks! !

⁣Sent from my phone, please excuse typos and briefness​

**Comment by pdebuyl** [comment]
I am glad to have contributed! For letting the world know: https://twitter.com/pdebuyl/status/915557754034679809

**Comment by GaelVaroquaux** [comment]
@pdebuyl : did you tag and push the tag? I am not seeing it (it seems like I forgot to do this in some previous releases)

**Comment by pdebuyl** [comment]
I did. I can see it in the web interface but I needed to refresh the page for the tag to appear.

**Comment by rougier** [comment]
Is there a 2017.1 release actually ? I can see the tag but not the release.

**Comment by pdebuyl** [comment]
I did not do a release. There is no systematic release (2016.1 does not have any). Should I do one?

**Comment by rougier** [comment]
I think so, this also how to get an updated DOI from Zenodo (that also handles versions now).

**Comment by GaelVaroquaux** [comment]
I made the release on the github interface.

**Pull request merged**